osp/serializers/fields.py

```diff
@@ -0,0 +1,63 @@
+from rest_framework import serializers
+
+from osp.models import (
+    Question, Choice, Checkbox, Dropdown,
+    Paragraph, ShortAnswer, Date, Time, FileUpload
+)
+from osp.serializers.question import QuestionSerializer
+
+class ChoiceSerializer(serializers.ModelSerializer):
+
+    class Meta(QuestionSerializer):
```

Comment:
You don't need to inherit `QuestionSerializer` here but rather in `ChoiceSerializer`